generators/generator-base-private.js

```diff
@@ -819,10 +819,16 @@ module.exports = class JHipsterBasePrivateGenerator extends Generator {
       this.debug(`File ${destination} ignored`);
       return Promise.resolved();
     }
-    return jhipsterUtils.renderContent(source, _this, _context, options).then(res => {
-      _this.fs.write(customDestination, res);
-      return customDestination;
-    });
+    return jhipsterUtils
```

Comment:
I see we use the async/await syntax elsewhere, can you change it so that we the newer one instead please? thanks

Reply:
Convert to async method is a breaking change.
While it doesn’t look so, blueprints still run it sync.
This is a bug, but let’s keep for 7.x cycle.